pcsx2/gui/AppHost.cpp

```diff
@@ -193,17 +195,19 @@ void Host::CheckForGSWindowResize()
 		return;
 
 	int width, height;
+	float scale;
```

Comment:
Good practice to initialize all 3.

Reply:
They all get initialized under the lock

IMO if you think you initialize a variable on all paths that you use it, it's more obvious to future readers and static analyzers if you don't initialize to a dummy value